main.cpp

```diff
@@ -56,10 +56,32 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    
+    int carrots = 4;
+    int potatoes = 6;
+    int onions = 2;
+
+    float time = 1.3f;
+    float bar = 4.6f;
+    float loopLength = 9.2f;
+
+    bool channelOn = true;
+    bool panFullyLeft = true;
+    bool channelLevelFull = true;
+    
+    double filterFrequency = 40.5669593;
+    double filterResonance = 1.5552231;
+    double filterBandwidth = 1.8999292;
+
+    char sampleBank = 'A';
+    char ampersand = 38;
+    char question = 63;
```

Comment:
`char` types are single-quoted alpha-numeric characters.  you're initializing your char types with integers, excluding line 76.
- [ ] revise.